tests/test_pydantic.py

```diff
@@ -4,6 +4,13 @@
 
 pydantic = pytest.importorskip("pydantic")
 
+try:
+    pydantic.version.version_info()
+    func_json = "model_dump_json"
+except:
```

Comment:
can we be explicit in what exception we are trying to catch? Is it `AttributeError`?

Reply:
added